browser-test/src/applicant/questions/text.test.ts

```diff
@@ -31,6 +33,11 @@ test.describe('Text question for applicant flow', () => {
       await logout(page)
     })
 
+    test.beforeEach(async () => {
+      const {page} = ctx
+      await disableFeatureFlag(page, 'north_star_applicant_ui')
+    })
```

Comment:
Unless you have the flag set to be enabled in `/server/conf/application.dev-browser-tests.conf` you shouldn't need this. The browser context is refresh in a beforeeach (index.tx) and the databases is dropped in an aftereach.

Reply:
if I remove this, I'm seeing the tests after the north star tests fail...

Reply:
Hmm. Ok. Maybe that index code isn't clearing something else after all. Appreciate you trying. Thanks